sig-scalability/thresholds.md

```diff
@@ -0,0 +1,96 @@
+# Kubernetes Scalability thresholds
+
+## Background
+
+Since 1.6 release Kubernetes officially supports 5000-node clusters. However,
+the question is what that actually means. As of early Q3 2017 we are in the
+process of defining set of performance-related SLIs ([Service Level Indicators])
+and SLOs ([Service Level Objectives]).
+
+However, no matter what SLIs and SLOs we have, there will always be some users
+coming and saying that their cluster is not meeting the SLOs. And in most cases
+it appears that the reason behind is that we (as developers) have silently
+assumed something (e.g. there will be no more than 10000 services in the
+cluster) and users were not aware of that.
+
+This document is trying to explicitly summarize all the limitations that we are
+aware of and state if we will try to relax them in the future or not.
+
+## Kubernetes thresholds
+
+We start with explicit definition of quantities and thresholds we assume are
+satisfied in the cluster. This is followed by an explanations for some of those.
+Important notes about the numbers:
+1. In most cases, exceeding these thresholds doesn’t mean that the cluster
+   fails over - it just means that its overall performance degrades.
+1. The thresholds obviously differ between different Kubernetes releases
+   (hopefully each of them is non-decreasing). The numbers we present are for
+   the current release (Kubernetes 1.7 release).
+1. There are a lot of factors that influence the thresholds, e.g. etcd version
+   or storage data format. For each of those we assume the default from the
+   release to avoid providing numbers for huge number of combinations of those.
+1. The “Current threshold” is representing the status of Kubernetes head. This
+   column should be snapshotted at every release to produce per-release
+   thresholds (and dedicated column for each release should then be added).
+
+| Quantity                            | Current threshold | Long term goal |
+|-------------------------------------|-------------------|----------------|
+| Total number of all objects         | 250000            | 1000000        |
+| Number of nodes                     | 5000              | 5000           |
+| Number of pods                      | 150000            | 500000         |
+| Number of pods per node<sup>1</sup> | 100               | 100            |
+| Number of pods per core<sup>1</sup> | 10                | 10             |
+| Number of namespaces (ns)           | 10000             | 100000         |
+| Number of pods per ns               | 15000             | 50000          |
+| Number of services                  | 10000             | 100000         |
+| Number of all services backends     | ?                 | 500000         |
+| Number of backends per service      | 5000              | 5000           |
+| Number of deployments per ns        | 20000             | 10000          |
+| Number of pods per deployment       | ?                 | 10000          |
```

Comment:
You will want to very clearly define what the Question marks mean.  Maybe even replace them with "TBD"?